src/components/Outline.js

```diff
@@ -50,7 +64,7 @@ class Outline extends Component {
 
     return functions
       .filter(func => func.value != "anonymous")
-      .map(this.renderFunction);
+      .map(func => this.renderFunction(func));
```

Comment:
Either approach is fine. We're generally prefer function references when there are no params`map(this.renderFunction)`, but we don't lint for it and either is fine.